bin/rialto-parachain/node/src/service.rs

```diff
@@ -483,11 +437,13 @@ pub async fn start_node(
 		 sync_oracle,
 		 keystore,
 		 force_authoring| {
+			let client2 = client.clone();
+			let block_import2 = block_import;
```

Comment:
Nit: I would rename these if we need them But maybe we can just clone on the spot where we use the client and the block_import.

Reply:
well, we just copy those functions here from Cumulus, so I didnt want to do a changes for future diffing